runtime/ecl/org.eclipse.rcptt.tesla.ecl.impl/src/org/eclipse/rcptt/tesla/ecl/internal/impl/commands/SelectorService.java

```diff
@@ -127,7 +126,7 @@ public IStatus service(Command command, IProcess context)
 				path += "%" + index + "%";
 			}
 			if (column != null) {
-				path += "#" + column + "#";
+				path += "#@" + column + "#@";
```

Comment:
Also please check if it is used in any other place and give version to @ulikcuc to verify on usual builds.